.travis.yml

```diff
@@ -3,39 +3,25 @@ sudo: false # don't need root access on Travis-CI (builds faster without sudo!)
 language: node_js # elm is installed from npm (see install below)
 node_js: node     # use "latest" version of Node.js
 
-env:
-  - ELM_VERSION=0.18.0 ELM_TEST=0.18.12
+# See https://github.com/cypress-io/cypress-example-kitchensink/blob/master/.travis.yml
+addons:
+  apt:
+    packages:
+      - libgconf-2-4
 
 cache:
   directories: # so subsequent builds run faster
     - elm-stuff/build-artifacts
     - elm-stuff/packages
```

Comment:
Seems like this is the 0.18 format - I'm not quite sure what we should be caching for 0.19. `elm-stuff` has subfolders `0.19.1` and `generated-code`.

Reply:
@jorgengranseth I'm not too concerned with _manually_ caching any directories.
The current build only takes 1 min 25 sec to run: 
https://travis-ci.org/dwyl/elm-datepicker/builds/650891968
<img width="921" alt="elm-datepicker-build" src="https://user-images.githubusercontent.com/194400/74593657-737ff880-5025-11ea-814a-792fb9ac7a1b.png">
This is a perfectly reasonable build time for an infrequently updated package. 💭

Reply:
Yeah, didn't seem like it would save much (if anything).

Reply:
We can investigate this later if it becomes tedious to wait for the build. 👍